private/buf/cmd/buf/command/format/format.go

```diff
@@ -183,15 +183,15 @@ type flags struct {
 	Output          string
 	Write           bool
 	// special
-	InputHashtag string
```

Comment:
`InputHashtag` is the same name across all commands, it's a special variable and matches with `bufcli.GetInputValue` naming, updating this would make `format` the one inconsistent command here - I'd like to keep this the same so you can `grep` for this in the future